meadow-client/entry.js

```diff
@@ -1,5 +1,5 @@
 import 'font-awesome/scss/font-awesome.scss';
-import './static/main.scss';
+import '../web-common/static/main.scss';
```

Comment:
I think there should probably still be a project-specific CSS file too.

Reply:
ok, I've added project specific main.scss for each project. One issue that is annoying is that `url(tx-arrows.svg)` had to be changed to `url(../../web-common/static/tx-arrows.svg)` which is a bit weird.

Reply:
I don't think `tx-arrows` should be common. It's definitely a plutus-specific thing.

Reply:
oh good point, that means that some of this shouldn't be in the shared css, but which parts exactly?

Reply:
I would be quite conservative here. I'd keep it as project local unless it only references standard tags or bootstrap classes.

For examples:

`^input` -> shared
`^h[1-5]` -> shared
`.wallet` -> plutus
`.wallet` -> action
`^.btn-sm` -> shared

Reply:
is this ok now @krisajenkins ?